app/src/main/kotlin/io/homeassistant/companion/android/launch/LaunchActivity.kt

```diff
@@ -42,9 +43,19 @@ import kotlinx.coroutines.launch
 import retrofit2.HttpException
 import timber.log.Timber
 
+private const val TO_SERVER = "go_to_server"
+
 @AndroidEntryPoint
 class LaunchActivity : AppCompatActivity(), LaunchView {
 
+    companion object {
+        fun newInstanceToSpecificServer(context: Context, serverUrl: String): Intent {
```

Comment:
I'm not sure at all about the naming of the variables in the `Lauch*` classes. So if you have suggestion be my guest.

Reply:
I wouldn't expect "goto server" from launching the app to open onboarding, but rather a server in the app (in which case you'd use the server ID). Why not something like `onboardServer` or `addServer`? That feels obvious to me...